build.sc

```diff
@@ -575,13 +575,12 @@ object scalajslib extends MillModule {
 
   override def generatedSources: Target[Seq[PathRef]] = Seq(generatedBuildInfo())
 
-  object api extends MillApiModule {
-    override def moduleDeps = Seq(main.api)
```

Comment:
Having a completely separated worker module has the side effect that now the Scala.js worker doesn't depend on Mill classes **at all**! :)